frontstage/views/sign_in.py

```diff
@@ -65,10 +87,10 @@ def login():
             return render_template("error.html", _theme='default', data={"error": {"type": "failed"}})
 
         data_dict_for_jwt_token = {
-            "refresh_token": token['refresh_token'],
-            "access_token": token['access_token'],
-            "scope": token['scope'],
-            "expires_at": token['expires_at'],
+            "refresh_token": oauth2_token['refresh_token'],
+            "access_token": oauth2_token['access_token'],
+            "scope": oauth2_token['scope'],
+            "expires_at": oauth2_token['expires_in'],
```

Comment:
The Oauth2 lib we used I believe was implementing some functionality to provide an expires at. Our system has since been built to decode this. In actual fact - the specs say expires_in.
https://tools.ietf.org/html/rfc6749#appendix-A.14